tests/run/t920.scala

```diff
@@ -1,20 +0,0 @@
-object Test {
```

Comment:
Not sure either, but I don't think that this test was added to scalac to test for that, cf https://issues.scala-lang.org/browse/SI-920 and https://github.com/scala/scala/commit/485a79aa79142c2740a64f8e7a83036067c7ee9c#diff-2a17630cbd90645d1f9a67af6e848c30
If it doesn't test anything useful then we can just drop it.